webapp/views.py

```diff
@@ -974,6 +1035,20 @@ def ensure_purchase_account():
     contains an auth token required for subsequent calls to the
     contract API.
     """
+    is_staging = flask.request.args.get("ua_staging", False)
+
+    api_url = flask.current_app.config["CONTRACTS_LIVE_API_URL"]
+
+    if is_staging:
+        print("Bah alors?????")
```

Comment:
I don't think these were meant to be left in the code. There are a few more below.

Reply:
:see_no_evil: I removed the console.log but forgot about these